### PR TITLE
feat: dark mode update #3

### PR DESCRIPTION
**Pull request by corquaid**
Description
---
Continuing Dark Mode implementation

- [x] Settings (Mining, Wallet, BaseNode, Docker)
- [x] Settings connected screens (Monero Auth, Aurora connection)
- [x] Password Prompt modals (Tari, Monero, combined)
- [x] Additional: ExpertView tabs, Main Layout scrollbar styling, Modal Backdrop
  
Motivation and Context
---
#36 partial

How Has This Been Tested?
---

**Screenshots**
_Mining settings_
Screenshot 2022-07-11 at 20 34 07
Screenshot 2022-07-11 at 20 34 30

_Wallet settings_
Screenshot 2022-07-11 at 20 34 52
Screenshot 2022-07-11 at 20 35 02

_BaseNode settings_
Screenshot 2022-07-11 at 20 35 11
Screenshot 2022-07-11 at 20 35 20

_Docker settings_
Screenshot 2022-07-11 at 20 35 36
Screenshot 2022-07-11 at 20 35 47

_Monero Auth modal_
Screenshot 2022-07-11 at 20 36 11
Screenshot 2022-07-11 at 20 36 20

_Connect Aurora modal_
Screenshot 2022-07-11 at 20 36 39
Screenshot 2022-07-11 at 20 36 50

_Password Prompts (couldn't get screenshots of Monero-only prompt)_
Screenshot 2022-07-11 at 20 37 23
Screenshot 2022-07-11 at 20 37 42
Screenshot 2022-07-11 at 20 37 59
Screenshot 2022-07-11 at 20 38 11

_ExpertView and HomePage scrollbar_
Screenshot 2022-07-11 at 20 47 49
Screenshot 2022-07-11 at 20 48 04

### CODE REVIEW

File: applications/launchpad/gui-react/src/components/Backdrop/index.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components'
 
 const Backdrop = styled.div<{ opacity?: number; borderRadius?: string }>`
```
Comment (tomaszantas) [inline applications/launchpad/gui-react/src/components/Backdrop/index.tsx:3]: NIT: I would suggest to add `$` to prop name so it won't be added to the DOM
```suggestion
const Backdrop = styled.div<{ opacity?: number; $borderRadius?: string }>`
```

Comment (tomaszantas) [reply applications/launchpad/gui-react/src/components/Backdrop/index.tsx:3]: Actually, we can do this also for the `opacity`: `$opacity?: number`.

**Review by tomaszantas — APPROVED** [review]
Looks good!

One suggestion in comments

File: applications/launchpad/gui-react/src/components/Backdrop/index.tsx (hunk 1 above)
Comment (corquaid) [reply applications/launchpad/gui-react/src/components/Backdrop/index.tsx:3]: I'll get it on the next PR.

**Pull request merged**